draft-wussler-openpgp-pqc.md

```diff
@@ -660,15 +660,17 @@ row "ECC-KEM" of {{tab-ecdh-cfrg-artifacts}}, {{tab-ecdh-nist-artifacts}}, and
 The operations `x25519Kem.encap()` and `x448Kem.encap()` are defined as
 follows:
 
- 1. Generate an ephemeral key pair {`v`, `V=vG`}, without any masking or
-    clamping as defined in [RFC7748]
+ 1. Generate an ephemeral key pair {`v`, `V=vG`} according to 
+    the procedure defined in RFC7748, that is, without any 
+    measures such as masking or clamping that are applied 
+    in previous OpenPGP specifications of this scheme.
 
- 2. Compute the shared point `X = vR` where `R` is the component public key
+ 3. Compute the shared point `X = vR` where `R` is the component public key
     `eccPublicKey` according to [RFC7748]
 
- 3. Set the output `eccCipherText` to `V`
+ 4. Set the output `eccCipherText` to `V`
 
- 4. Set the output `eccKeyShare` to `X` in native format
+ 5. Set the output `eccKeyShare` to `X` in native format
```

Comment:
Now the list seems to go from 1 then 3,4,5. Is the 2 missing?

Reply:
FYI this does not matter with markdown, but I agree it's nice to keep tidy